app/src/main/java/woowacourse/movie/activity/MainActivity.kt

```diff
@@ -27,29 +32,59 @@ class MainActivity : AppCompatActivity(), NavigationBarView.OnItemSelectedListen
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        PermissionManager.requestNotificationPermission(this, requestPermissionLauncher)
+        requestNotificationPermission()
         findViewById<BottomNavigationView>(R.id.bnv_main).setOnItemSelectedListener(this)
-        supportFragmentManager.beginTransaction().add(R.id.fl_main, BookHistoryFragment()).commit()
+        supportFragmentManager.beginTransaction().add(
+            R.id.fl_main,
+            fragments.getOrPut(R.id.page_book_history) { BookHistoryFragment() }
+        ).commit()
```

Comment:
```suggestion
        supportFragmentManager.commit {
            add(
                 R.id.fl_main,
                 fragments.getOrPut(R.id.page_book_history) { BookHistoryFragment() }
             )
         }
```

fragmentManager 에서 제공되는 `commit` 을 활용하면 transaction 선언을 생략할 수 있으니 참고 부탁드려요~!